pyogrio/geopandas.py

```diff
@@ -181,6 +181,9 @@ def write_dataframe(
 
     Parameters
     ----------
+    df : GeoDataFrame
+        The data to write. For attribute columns of the "object" dtype, all values will
+        be converted to strings to be written to the output file.
```

Comment:
```suggestion
        be converted to strings to be written to the output file, except None and np.nan, which will be set to NULL in the output file.
```

(not sure if the distinction is important, but clarifies that `None` is set to null rather than `"None"`)

Reply:
Ok, sounds good.